ibm/service/vpc/data_source_ibm_is_image.go

```diff
@@ -289,10 +292,12 @@ func dataSourceIBMISImageRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
-func imageGetByName(d *schema.ResourceData, meta interface{}, name, visibility string) error {
+func imageGetByName(context context.Context, d *schema.ResourceData, meta interface{}, name, visibility string) diag.Diagnostics {
 	sess, err := vpcClient(meta)
```

Comment:
are we not doing `vpcClient, err := meta.(conns.ClientSession).VpcV1API()` ? Does it have any counter impact?

Reply:
```
func vpcClient(meta interface{}) (*vpcv1.VpcV1, error) {
	sess, err := meta.(conns.ClientSession).VpcV1API()
	return sess, err
}
```

both are same